lib/global_registry_bindings/options/entity_options_parser.rb

```diff
@@ -11,8 +11,7 @@ def initialize(model_class)
         def defaults
           {
             binding: :entity,
-            id_column: :global_registry_id,
-            mdm_id_column: nil,
+            id_column: :global_registry_id, mdm_id_column: nil, fingerprint_column: nil,
```

Comment:
This is a code style thing, but I don't like having multiple entries on one row if the rest of the rows are single entries. 🤷‍♂️

Reply:
lol the things we do to keep rubocop happy...